src/main/kotlin/org/rust/lang/core/psi/ext/RsPat.kt

```diff
@@ -28,7 +26,7 @@ val RsPat.isIrrefutable: Boolean
             pat.patBinding.isIrrefutable
         is RsPatConst ->
             (pat.expr as? RsPathExpr)?.path?.isIrrefutable ?: false
-        is RsPatRange ->
+        is RsPatRange, is RsPatSlice ->
```

Comment:
What's about the following case:
```rust
fn main() {
    if let [a@ ..] = [1, 2, 3] {
        println!("{:?}", a);
    }
}
```

Looks like it's irrefutable slice pattern

Reply:
Fixed.